src/resources/Microsoft.Authorization/policyDefinitions/Deny-VMUnmanagedDisk.json

```
@@ -0,0 +1,67 @@
+{
+    "name": "Deny-VM-with-unmanaged-disk",
+    "apiVersion": "2021-06-01",
+    "scope": "null",
+    "properties": {
+        "policyType": "Custom",
+        "mode": "all",
+        "displayName": "Deny virtual machines and virtual machine scalesets not using managed disk",
+        "description": "Deny virtual machines not using managed disk. It checks the managedDisk property on virtual machine OS, and Datadisk fields.",
```

```suggestion
        "description": "Deny Virtual Machines and Virtual Machine Scale Sets from not using Managed Disks. It checks the managedDisk property on Virtual Machine & Virtual Machine Scale Sets OS, and Data Disk fields.",
```

This also isnt true about data disks in its current form